### PR TITLE
🧹  Clean up API reports in ldap and msgraph catalog modules

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -136,3 +136,6 @@ site
 
 # e2e tests
 cypress/cypress/*
+
+# Possible leftover from build:api-reports
+tsconfig.tmp.json
```
Comment (Rugvip) [inline .gitignore:141]: That's a bug in that case, did you end up with it?

**Review by jhaals — APPROVED** [review]
Nice spring cleaning!

**Review by Rugvip — APPROVED** [review]
🎉

File: .gitignore (hunk 1 above)
Comment (freben) [reply .gitignore:141]: Yeah. Probably after ctrl+C or so

Comment (Rugvip) [reply .gitignore:141]: The intention is to handle anything but `kill -9`

**Pull request merged**